hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/spi/impl/TestOzoneManagerServiceProviderImpl.java

```diff
@@ -147,6 +151,14 @@ public void testUpdateReconOmDBWithNewSnapshot() throws Exception {
         .get("/sampleVol/bucketOne/key_two"));
   }
 
+  private static SCMNodeDetails getReconNodeDetails() {
+    SCMNodeDetails.Builder builder = new SCMNodeDetails.Builder();
+    builder.setSCMNodeId("Recon");
+    builder.setDatanodeProtocolServerAddress(
+        InetSocketAddress.createUnresolved("127.0.0.1", 9888));
+    return builder.build();
+  }
```

Comment:
It would be nice to extract this to a public test helper class (e.g. `CommonUtils`?) and reuse in other unit tests.